server.js

```diff
@@ -2,49 +2,74 @@
 
 var request = require('request')
 var express = require('express')
+var xml2js = require('xml2js')
+var coberturaPath = '/jenkins/cobertura/'
+var jacocoPath = '/jenkins/jacoco/'
 
 var app = express()
+var badgeColor = function(cov) {
+	if (cov < 20) {
+		return 'red'
+	} else if (cov < 80) {
```

Comment:
In the future we may want to see if we can pull the coverage threshold values (If they're set) from the Jenkin job's config file.